pymc/logprob/abstract.py

```diff
@@ -153,3 +155,29 @@ def __init__(self, scalar_op, *args, **kwargs):
 
 
 MeasurableVariable.register(MeasurableElemwise)
+
+
+@multipledispatch(Op, Op)
+def _kl_div(
+    q_dist: Op,
+    p_dist: Op,
+    q_inputs: List[TensorVariable],
+    p_inputs: List[TensorVariable],
+) -> TensorVariable:
+    raise NotImplementedError(
+        f"KL Divergence is not implemented for {q_dist}, {p_dist}"
+    )
+
+
+def _kl_div_helper(
+    q_rv: TensorVariable,
+    p_rv: TensorVariable,
+) -> TensorVariable:
+    kl = _kl_div(
+        q_rv.owner.op,
+        p_rv.owner.op,
+        q_inputs=q_rv.owner.inputs,
```

Comment:
Perhaps pass the node instead of inputs. Allows stuff like op.dist_params(node) and op.size_param(node) inside the dispatch functions. Not sure though